CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Changed
+- **BREAKING:** Rename the package from `eth-json-rpc-middleware` to `@metamask/eth-json-rpc-middleware` ([#180](https://github.com/MetaMask/eth-json-rpc-middleware/pull/180))
+- Change all middleware request and response types to `unknown` ([#183](https://github.com/MetaMask/eth-json-rpc-middleware/pull/183))
```

Comment:
I'm _pretty sure_ this isn't a breaking change, but not 100% sure. I can update this retroactively if we decide it should be breaking (cc @mcmire I'd be interested to hear your take on this).

My reasoning is that the `unknown` types are the inputs, so this change makes them strictly more flexible. The same goes for the addition of the optional `skipCache` property.